.github/workflows/rename_project.yml

```diff
@@ -6,7 +6,7 @@ permissions: write-all
 
 jobs:
   rename-project:
-    if: ${{ !contains (github.repository, '/cnd-python-template') }}
+    if: ${{ !contains (github.repository, '/cnd-etl-template') }}
```

Comment:
Is this supposed to happen (since we're in cnd-python-template repo I would assume the check of repo name is also about that)?